src/System.Globalization/tests/RegionInfo/RegionInfoTests.Properties.cs

```diff
@@ -2,19 +2,27 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
 using System.Linq;
 using Xunit;
 
 namespace System.Globalization.Tests
 {
-    public class RegionInfoPropertyTests
+    public class RegionInfoPropertyTests : RemoteExecutorTestBase
     {
         [Fact]
         public void CurrentRegion()
         {
-            RegionInfo ri = new RegionInfo(new RegionInfo(CultureInfo.CurrentCulture.Name).TwoLetterISORegionName);
-            Assert.True(RegionInfo.CurrentRegion.Equals(ri) || RegionInfo.CurrentRegion.Equals(new RegionInfo(CultureInfo.CurrentCulture.Name)));
-            Assert.Same(RegionInfo.CurrentRegion, RegionInfo.CurrentRegion);
+            RemoteInvoke(() =>
+            {
+                CultureInfo.CurrentCulture = new CultureInfo("en-US");
+
+                RegionInfo ri = new RegionInfo(new RegionInfo(CultureInfo.CurrentCulture.Name).TwoLetterISORegionName);
+                Assert.True(RegionInfo.CurrentRegion.Equals(ri) || RegionInfo.CurrentRegion.Equals(new RegionInfo(CultureInfo.CurrentCulture.Name)));
+                Assert.Same(RegionInfo.CurrentRegion, RegionInfo.CurrentRegion);
+
+                return SuccessExitCode;
+            }).Dispose();
```

Comment:
Yeah, I can do that.  I miss-remembered CurrentCulture as a process wide thing instead of thread wide and was worried that I was going to be interacting poorly with other tests in racy cases.